src/main/java/org/jitsi/jicofo/rest/Health.java

```diff
@@ -217,9 +225,21 @@ static synchronized void getJSON(
             }
             else
             {
+                HealthChecksMonitor monitor = null;
+                if (focusManager.isHealthChecksDebugEnabled())
+                {
+                    monitor = new HealthChecksMonitor();
+                    monitor.start();
+                }
+
                 check(focusManager);
 
                 status = cacheStatus(HttpServletResponse.SC_OK);
+
+                if (monitor != null)
+                {
+                    monitor.stop();
```

Comment:
this needs to by in try{ check(...); } finally { monitor.stop()} otherwise those will be leaking when check throws (see the outer try catch bloc).